motion_controller:
  ros__parameters:
    angular_pid:
      kp: 0.8
      ki: 0.01
      kd: 0.05
      max_output: 1.0
      min_output: -1.0
    linear_pid:
      kp: 0.5
      ki: 0.001
      kd: 0.02
      max_output: 0.5
      min_output: 0.0